docs/content/en/docs-dev/user-guide/configuration-reference.md

```diff
@@ -19,6 +19,8 @@ spec:
 
 | Field | Type | Description | Required |
 |-|-|-|-|
+| name | string | The application name. | Yes if you set the application through the application configuration file |
```

Comment:
```suggestion
| name | string | The application name. | Yes (if you want to create PipeCD application through the application configuration file) |
```
nits, and same for the others.

Reply:
Looks good. But currently others are in the form of this. So let me work on it as a separate PR.
![image](https://user-images.githubusercontent.com/19730728/146146700-25b21353-93fe-4d06-a57a-c5add344257d.png)

Reply:
Okay, for the sentence, let me borrow yours 👍

Reply:
Thanks for clarifying 👍